docs/source/getting-started/index.rst

```diff
@@ -9,7 +9,7 @@ how to use MLflow to solve a particular task.
 Guidance on Running Tutorials
 -----------------------------
 
-If you are new to MLflow and have never interfaced with the MLflow Tracking Server, we highly encourage you to head on over to quickly read the guide below. It 
+If you are new to MLflow and have never interfaced with the MLflow Tracking Server, we highly encourage you to head on over to quickly **read the guide below**. It 
```

Comment:
nit: Can we add more inline links to our technical terms like [MLflow Tracking Server](https://mlflow.org/docs/latest/tracking/server.htmll), [MLflow fluent API](https://mlflow.org/docs/latest/tracking/tracking-api.html)? It provides answers when user faces question "what this term means?" while not needing repeating explanation.

Reply:
Added a few links where they make sense